<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>174</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>174</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>174</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="lableTitle">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>23</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelBody">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>NEW YORK, Nov. 14, 2020 /PRNewswire/ -- Rosen Law Firm, a global investor rights law firm, reminds purchasers of the securities of Turquoise Hill Resources Ltd. (NYSE: TRQ) between July 17, 2018 and July 31, 2019, inclusive (the &quot;Class Period&quot;), of the important December 14, 2020 lead plaintiff deadline in the securities class action. The lawsuit seeks to recover damages for Turquoise Hill investors under the federal securities laws.
To join the Turquoise Hill class action, go to http://www.rosenlegal.com/cases-register-1971.html or call Phillip Kim, Esq. toll-free at 866-767-3653 or email pkim@rosenlegal.com or cases@rosenlegal.com for information on the class action.
According to the lawsuit, throughout the Class Period and regarding the development of the Oyu Tolgoi copper-gold mine in Mongolia, defendants made false and/or misleading statements and/or failed to disclose that: (1) the stability issues were much more severe than represented and called into question the design of the mine, the projected cost and timing of production; (2) the publicly disclosed estimates of the cost, date of completion and dates for production from the underground mine were not achievable; (3) the &quot;challenging ground conditions&quot; were much more severe than defendants represented, and in fact made it impossible for Turquoise Hill and Rio Tinto to achieve those estimates; (4) the development capital required for the underground development of Oyu Tolgoi would cost substantially more than a billion dollars over what Turquoise Hill and Rio Tinto had represented; (5) Turquoise Hill would require additional financing and/or equity to complete the project; (6) the progress of underground development and of Oyu Tolgoi was not proceeding as planned; and (7) the &quot;key risks&quot; had not been &quot;well understood and managed&quot; but had placed the project schedule and cost into severe jeopardy. When the true details entered the market, the lawsuit claims that investors suffered damages.
A class action lawsuit has already been filed. If you wish to serve as lead plaintiff, you must move the Court no later than December 14, 2020. A lead plaintiff is a representative party acting on behalf of other class members in directing the litigation. If you wish to join the litigation, go to http://www.rosenlegal.com/cases-register-1971.html or to discuss your rights or interests regarding this class action, please contact Phillip Kim, Esq. of Rosen Law Firm toll free at 866-767-3653 or via e-mail at pkim@rosenlegal.com or cases@rosenlegal.com.
NO CLASS HAS YET BEEN CERTIFIED IN THE ABOVE ACTION. UNTIL A CLASS IS CERTIFIED, YOU ARE NOT REPRESENTED BY COUNSEL UNLESS YOU RETAIN ONE. YOU MAY RETAIN COUNSEL OF YOUR CHOICE. YOU MAY ALSO REMAIN AN ABSENT CLASS MEMBER AND DO NOTHING AT THIS POINT. AN INVESTOR'S ABILITY TO SHARE IN ANY POTENTIAL FUTURE RECOVERY IS NOT DEPENDENT UPON SERVING AS LEAD PLAINTIFF.
Follow us for updates on LinkedIn: https://www.linkedin.com/company/the-rosen-law-firm, on Twitter: https://twitter.com/rosen_firm or on Facebook: https://www.facebook.com/rosenlawfirm/.
Rosen Law Firm represents investors throughout the globe, concentrating its practice in securities class actions and shareholder derivative litigation. Rosen Law Firm was Ranked No. 1 by ISS Securities Class Action Services for number of securities class action settlements in 2017. The firm has been ranked in the top 3 each year since 2013. Rosen Law Firm has achieved the largest ever securities class action settlement against a Chinese Company. Rosen Law Firm's attorneys are ranked and recognized by numerous independent and respected sources. Rosen Law Firm has secured hundreds of millions of dollars for investors. Attorney Advertising. Prior results do not guarantee a similar outcome.
Contact Information:
      Laurence Rosen, Esq.
      Phillip Kim, Esq.
      The Rosen Law Firm, P.A.
      275 Madison Avenue, 40th Floor
      New York, NY 10016
      Tel: (212) 686-1060
      Toll Free: (866) 767-3653
      Fax: (212) 202-3827
      lrosen@rosenlegal.com
      pkim@rosenlegal.com
      cases@rosenlegal.com
      www.rosenlegal.com
SOURCE Rosen Law Firm, P.A.ï¿¼
Related Links
www.rosenlegal.com</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lableStock">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nasdaq</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelTime">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>156</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>68</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>12:00 PM
2020/02/02</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
